kubernetes/crd/domain-crd.yaml

```diff
@@ -347,6 +347,145 @@ spec:
                     name:
                       type: string
                 type: array
+              fluentdSpecification:
+                description: Automatic fluentd sidecar injection. If specified, the
```

Comment:
We should mention that there is an option to inject sidecars for the introspection pod.

Reply:
Added and corrected other reference to exporter